nmsg/kafkaio.c

```diff
@@ -283,6 +348,11 @@ _kafka_init_consumer(kafka_ctx_t ctx, rd_kafka_conf_t *config)
 		}
 	}
 
+	if (!_kafka_process_config(ctx, config)) {
```

Comment:
I think the configuration file should probably override (come after) all the config opts that we set manually via `_kafka_config_set_option()` EXCEPT for the ones when there's a group id specified on the command line ("group.id" and "auto.offset.reset")... so maybe lines 336-349 should come after this one.